obspy/taup/tau.py

```diff
@@ -165,6 +165,56 @@ def copy(self):
         return self.__class__(super(Arrivals, self).copy(),
                               model=self.model)
 
+    def timeplot(self, phases, ax=None, show=True):
+        """
+        Method to plot the travel times of arrivals class, if any have
+        been calculated.
+
+        :param phases: Phases to plot.
+        :type phases: list of str
+        :param ax: Axes to plot to. If not given, a new figure with an axes
+            will be created. Must be a polar axes for the spherical plot and
+            a regular one for the Cartesian plot.
+        :type ax: :class:`matplotlib.axes.Axes`
+        :param show: Show the plot.
+        :type show: bool
+
+        :returns: The (possibly created) axes instance.
+        :rtype: :class:`matplotlib.axes.Axes`
+
+        """
+        import matplotlib.pyplot as plt
+        cmap = plt.cm.Dark2
+        colors = cmap(np.linspace(0, 1, len(phases)))
+        if not self:
+            raise ValueError("Can only plot calculated travel times.")
+
+        # create an axis/figure, if there is none, yet:
+        if not ax:
+            plt.figure(figsize=(10, 10))
+            ax = plt.subplot(111)
```

Comment:
Ideally, this should all be done with clean API, not relying on current figure / current axes. I.e...

```python
if ax:
    fig = ax.figure
else:
    fig, ax = plt.subplots()
```

And then in the following all plot commands on `ax` not using pylab (current axes), so avoiding things like `plt.plot(..)` and using `ax.plot()` instead.